esp-hal/src/rtc_cntl/mod.rs

```diff
@@ -517,19 +515,8 @@ pub enum RwdtStage {
 /// RTC Watchdog Timer.
 pub struct Rwdt;
```

Comment:
```suggestion
pub struct Rwdt(());
```

I guess we should make it something like this, so that users can't just create them from thin air

Reply:
`#[non_exhaustive]` is the solution for that

Reply:
We should fix it here whilst we're here.